### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Review by wborn — APPROVED** [review]

**Review by ghys — APPROVED** [review]

**Comment by ghys** [comment]
My approval was "on behalf of @openhab/webui-maintainers" it seems but I'll let @kaikreuzer or @lolodomo merge this one!
I think everyone agrees.

**Pull request merged**
